apis/v1alpha1/instrumentation_types.go

```diff
@@ -137,6 +137,9 @@ type InstrumentationStatus struct {
 // +kubebuilder:resource:shortName=otelinst;otelinsts
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:printcolumn:name="OTEL-Endpoint",type="string",JSONPath=".spec.exporter.endpoint"
+// +kubebuilder:printcolumn:name="Sample-Type",type="string",JSONPath=".spec.sampler.type"
+// +kubebuilder:printcolumn:name="Sample-Rate",type="string",JSONPath=".spec.sampler.argument"
```

Comment:
Rename this to `Sampler Arg`. Can there be a space in the name?

Reply:
seems to work :)